include/Note.h

```diff
@@ -246,7 +259,10 @@ class LMMS_EXPORT Note : public SerializingObject
 	panning_t m_panning;
 	TimePos m_length;
 	TimePos m_pos;
-	DetuningHelper * m_detuning;
+	DetuningHelper * m_detuning = nullptr;
+
+	// The type of this note
+	Type m_type = Type::RegularNote;
```

Comment:
This comment is fairly unnecessary as it is mostly self explanatory. I would suggest removing it and the spaces between ``m_type`` and the preceding ``m_detuning`` variable. For reference, [NL.1](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#nl1-dont-say-in-comments-what-can-be-clearly-stated-in-code) from the guidelines.